pkg/apis/kops/componentconfig.go

```diff
@@ -930,6 +930,10 @@ type CertManagerConfig struct {
 	// Default: false
 	Enabled *bool `json:"enabled,omitempty"`
 
+	// External Tell kops that cert manager is deployed externally.
+	// Default: false
+	External *bool `json:"external,omitempty"`
```

Comment:
Let me check if this naming is consistent with other structs where we allow this.

Reply:
I think some other time we discussed about "Managed".

Reply:
you mean setting "managed" to false?

Reply:
Example: https://github.com/kubernetes/kops/blob/ba3914a30b41a986f687f25d51cff36951d03070/pkg/apis/kops/ntpconfig.go#L23

Reply:
Changed the PR to do it the same way